jarvis.py

```diff
@@ -16,6 +16,22 @@
 def speak(audio):

```

Comment:
forget to import screen_brightness_control without this you can't use these function
so add this line  (import screen_brightness_control as sbc)
brfore that go to cmd and import  screen_brightness_control